frame/contracts/src/lib.rs

```diff
@@ -782,6 +787,9 @@ pub mod pallet {
 			/// The code hash that was delegate called.
 			code_hash: CodeHash<T>,
 		},
+
+		/// Some funds has been held as storage deposit.
+		StorageDepositHeld { who: T::AccountId, amount: BalanceOf<T> },
```

Comment:
These two fields have been imported from `Event::Reserved`, do we want to add something else, like `code_hash`?